src/app/home/guards/has-role.guard.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+import {
+  CanActivate
+} from '@angular/router';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class HasRoleGuard implements CanActivate {
+  canActivate() {
+    if (localStorage.getItem('role') == 'admin') return true;
```

Comment:
Need to update to check for any role not just admin.

TBD - On this implementation